meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Don't use arrow functions in a react callback like that: https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36